### ISSUE TITLE
onpress for axis labels?

### ISSUE THREAD

**Issue by vjsingh** [JesperLekland/react-native-svg-charts#190]
Is it possible to pass in a callback for when a certain label on an axis is pressed? I have a StackedBarChart and I'd like to do something when the labels on the x-axis are pressed.

**Comment by JesperLekland** [comment]
yes, the axes supports complex data structures as well (`svg` prop with `onPress` for each item)

**Comment by vjsingh** [comment]
@JesperLekland Sorry I didn't respond to this yet. Thank you, passing in an onPress to the axis svgs seems to be working. 

It appears that the touch target however is the text, which makes for a finicky touch target. Would there be an easy way to pass in a custom rectangle that has text on it (to make the touch target bigger) or is that something I would have to do custom myself?

**Comment by JesperLekland** [comment]
Great to hear that it worked. 

Unfortunately there's no way of passing in any other shapes as this would break the layout. The closest solution I can think of is to create your own transparent component on top of the xAxis with rectangles positioned the same way the text is. You can look at the [XAxis source](https://github.com/JesperLekland/react-native-svg-charts/blob/dev/src/x-axis.js) for inspiration.

**Comment by vjsingh** [comment]
gotcha, that seems like a good solution. Thank you!
